redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewFlexContainer.kt

```diff
@@ -71,7 +71,8 @@ internal class UIViewFlexContainer(
   }
 
   override fun onEndChanges() {
-    value.setNeedsLayout()
+    value.invalidateIntrinsicContentSize() // Tell the enclosing view that our size changed.
+    value.setNeedsLayout() // Update layout of subviews.
```

Comment:
Without this, the TreehouseUIView doesn’t change sizes when new items are added to a flex container.

Reply:
Usually, the way to [force a layout pass](https://developer.apple.com/documentation/uikit/uiview/1622482-layoutsubviews) is to call `setNeedsLayout()` followed by `layoutIfNeeded()`--does that not suffice here?

Reply:
UIKit Autolayout and SwiftUI depends on `intrinsicContentSize` for sizing changes, while UIKit frame layout relies on needsLayout, we probably need both to support all cases.

Reply:
How can we test both scenarios?

Reply:
I can do a follow-up to test a flex layout inside an enclosing layout. It’ll look a lot like the test added to this PR, but it’ll use a flex layout as the child of the TreehouseView